locales/es/es.json

```diff
@@ -328,7 +329,7 @@
     "Luxembourg": "Luxemburgo",
     "Macao": "Macao",
     "Macedonia": "Macedonia",
-    "Macedonia, the former Yugoslav Republic of": "Macedonia, the former Yugoslav Republic of",
+    "Macedonia, the former Yugoslav Republic of": "Macedonia, ex República Yugoslava de",
```

Comment:
you do two translations ?

Reply:
I only translate `locales/es/es.json` based in `source/en.json` and in this one the countries was repeated.